bodo/pandas/plan_operators.py

```diff
@@ -0,0 +1,227 @@
+from bodo.pandas.parquet import get_pandas_schema
```

Comment:
These nodes will change substantially. But is there a reason to have wrappers in regular Python vs just using the Cython nodes (which are wrappers on C++)?

https://github.com/bodo-ai/Bodo/pull/344/files

Reply:
It isn't clear to me if what you are suggesting is possible yet.  It definitely isn't possible if we are just talking LogicalOperators.  If you take the transitve closure of all types defined or used in LogicalOperators then perhaps but at a minimum it doesn't assemble cleanly as I think the duckdb objects generated for the same Python code are context dependent.